.clang-format

```diff
@@ -82,17 +82,20 @@ IncludeBlocks: Regroup
 #finally include all system headers - characterized as headers wrapped with '< >'
 IncludeCategories:
   - Regex:           '(<.*>)'
+    Priority:        4
+    SortPriority:    4
+  - Regex:           '("[^\(sst\)].*")'
     Priority:        3
     SortPriority:    3
-  - Regex:           '("[^\(sst\)].*")'
-    Priority:        2
-    SortPriority:    2
-  - Regex:           '("sst/core/.*")'
+  - Regex:           '("sst/core/testElements/.*")'
```

Comment:
Why do we need to single this specific folder out?

Reply:
Even though testElements is part of the core repository, it should be treated like an element library.  Hence, its header files should come before other core header files.